src/pages/components/content-item.mdx

```diff
@@ -1,10 +1,8 @@
 ---
 title: Content item
 description:
-  The Content item is a basic-level content component that acts as a primary content unit, and it is used inside a
-  content group to build web experiences on IBM dotcom pages. It can encapsulate basic content (a heading, a
-  description, a media element – image or video – and a CTA) and brings them all together into a content component which
-  solves a specific problem for the user.
+  The content item is a basic-level content component that is used inside a
+  content group to build web experiences on IBM dotcom pages. It can encapsulate media and text content and allow for more variation in an editorial experience.
```

Comment:
```suggestion
  The content item component is a basic-level content component that is used inside a
  content group to build web experiences on IBM.com pages. It can encapsulate media and text content and allows for more variation in an editorial experience.
```

Reply:
Additional suggestion:

..."that is used inside _higher level content components_ to build web experiences..." rather than only saying content group. I like how you called out the content section and content group further down the page guidance.